main.nf

```diff
@@ -1852,12 +1852,12 @@ process annotate_vep {
 			--plugin CADD,$params.CADD \\
 			--plugin LoFtool \\
 			--plugin MaxEntScan,$params.MAXENTSCAN,SWA,NCSS \\
-			--plugin dbNSFP,/fs1/resources/ref/hg38/annotation_dbs/dbnsfp/dbNSFP4.3a_grch38.gz,transcript_match=1,REVEL_score,REVEL_rankscore \\
+			--plugin dbNSFP,$params.DBNSFP,transcript_match=1,REVEL_score,REVEL_rankscore \\
 			-custom $params.GNOMAD_EXOMES,gnomADe,vcf,exact,0,AF_popmax,AF,popmax \\
 			-custom $params.GNOMAD_GENOMES,gnomADg,vcf,exact,0,AF_popmax,AF,popmax \\
 			-custom $params.GNOMAD_MT,gnomAD_mt,vcf,exact,0,AF_hom,AF_het \\
-			-custom $params.PHYLOP \\
-			-custom $params.PHASTCONS
+			-custom $params.PHYLOP,phyloP100way,bigwig \\
+			-custom $params.PHASTCONS,phastCons,bigwig
```

Comment:
What are those two changes? Should they be documented in the changelog?

Reply:
These where previously defined in the config file, i.e. part of the "$params.PHASTCONS" variable. So the main.nf + nextflow.conf pair should produce the same results after this update, if done correctly.

But yes, makes sense to specify this change in the changelog

Reply:
Ohh. I see it now.